sub_accounts/client_sub_account.go

```diff
@@ -50,18 +50,23 @@ type AccountUtilizationSettings struct {
 }
 
 type SubAccount struct {
-	AccountId              int32                      `json:"accountId"`
-	Email                  string                     `json:"email"`
-	AccountName            string                     `json:"accountName"`
-	Flexible               bool                       `json:"isFlexible"`
-	ReservedDailyGB        float32                    `json:"reservedDailyGB"`
-	MaxDailyGB             float32                    `json:"maxDailyGB"`
-	RetentionDays          int32                      `json:"retentionDays"`
-	Searchable             bool                       `json:"searchable"`
-	Accessible             bool                       `json:"accessible"`
-	DocSizeSetting         bool                       `json:"docSizeSetting"`
-	SharingObjectsAccounts []SharingAccount           `json:"sharingObjectsAccounts"`
-	UtilizationSettings    AccountUtilizationSettings `json:"utilizationSettings"`
+	AccountId               int32                      `json:"accountId"`
+	Email                   string                     `json:"email"`
+	AccountName             string                     `json:"accountName"`
+	Flexible                bool                       `json:"isFlexible"`
+	ReservedDailyGB         float32                    `json:"reservedDailyGB"`
+	MaxDailyGB              float32                    `json:"maxDailyGB"`
+	RetentionDays           int32                      `json:"retentionDays"`
+	Searchable              bool                       `json:"searchable"`
+	Accessible              bool                       `json:"accessible"`
+	DocSizeSetting          bool                       `json:"docSizeSetting"`
+	SharingObjectsAccounts  []SharingAccount           `json:"sharingObjectsAccounts"`
+	UtilizationSettings     AccountUtilizationSettings `json:"utilizationSettings"`
+	SnapSearchRetentionDays int32                      `json:"snapSearchRetentionDays"`
+	IsCapped                bool                       `json:"isCapped"`
```

Comment:
I dont see `IsCapped` `SharedGB` `TotalTimeBasedDailyGB` `IsOwner` used anywhere In the code. am i missing something? Do we have to add them?